frontend/src/layout/onboarding/tour.js

```diff
@@ -0,0 +1,193 @@
+import React from 'react'
+import Tour from 'reactour'
+import { useValues, useActions } from 'kea'
+import { onboardingLogic, TourType } from './onboardingLogic'
+import { Button } from 'antd'
+
+export function PGTour() {
```

Comment:
Consider `PgTour`
Eg `HTML` -> `Html`
The recommendation mainly comes from the perspective of consistency. It is annoying having to remember what parts of an acronym are capitalized, which parts aren't, and it usually becomes mixed up throughout the codebase (eg some devs use `Html` others use `HTML`)

Source: _Framework Design Guidelines_ (happy to quote the entire section for reference)